site/docs/components/file-drop-zone/examples.mdx

```diff
@@ -0,0 +1,46 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+  <LivePreview componentName="file-drop-zone" exampleName="Default">
+
+    ### FileDropZone
+
+    This component allows users to upload files they've selected from the native file browser interface or dragged-and-dropped — with the files automatically uploaded to your web application.
+    The message in the File drop zone can be customized to suit your needs.
+
+    Note that the banner is not part of the component, and is in the example as an illustration only.
+    Information about files you upload to this example will be stored on our UI Toolkit servers.
+
+  </LivePreview>
+  <LivePreview componentName="file-drop-zone" exampleName="Error">
+
+    ### Error
+
+    Inform users when a file cannot be uploaded successfully. Error messages should provide feedback to guide the user resolve the issue.
+
+  </LivePreview>
+  <LivePreview componentName="file-drop-zone" exampleName="Success">
+
+    ### Success
+
+    Inform users when files successfully upload. Success messages should reassure users that the action has been completed.
+    
+  </LivePreview>
+  <LivePreview componentName="file-drop-zone" exampleName="Disabled">
+
+    ### Disabled
+    
+    The File Drop Zone component can be disabled, with no resultant action when users drop a file onto the component.
```

Comment:
You should add guidance that mentions to make sure disabled is passed to the trigger too.